Sprint-3/revise/implement/count.test.js

```diff
@@ -15,3 +15,29 @@
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
```

Comment:
note the case-sensitive part, so the tests should cover a mix of capital and small letters